iosxr_setup.py

```diff
@@ -170,9 +167,9 @@ def setup_networking(self, host_ip=None, gateway=None, allow_root_login=False):
         # This will prevent users from needing to supply another Vagrantfile or editing /etc/resolv.conf manually
         # Doing this in xrnns because the syncing of /etc/netns/global-vrf/resolv.conf to
         # /etc/resolv.conf requires 'ip netns exec global-vrf bash'.
-        self.send_xrnns('echo "# Cisco OpenDNS IPv4 nameservers" >> /etc/resolv.conf')
-        self.send_xrnns('echo "nameserver 208.67.222.222" >> /etc/resolv.conf')
-        self.send_xrnns('echo "nameserver 208.67.220.220" >> /etc/resolv.conf')
+        self.send_xrnns("echo '# Cisco OpenDNS IPv4 nameservers' >> /etc/resolv.conf")
```

Comment:
tee -a might be better in general - it gets around all these permission
worries of doing
stuff in 2 steps...

On 1 July 2016 at 13:22, Glenn Matthews <notifications@github.com> wrote:

> In iosxr_setup.py
> <https://github.com/ios-xr/iosxrv-x64-vbox/pull/3#discussion_r69330827>:
>
> > @@ -170,9 +167,9 @@ def setup_networking(self, host_ip=None, gateway=None, allow_root_login=False):
> >          # This will prevent users from needing to supply another Vagrantfile or editing /etc/resolv.conf manually
> >          # Doing this in xrnns because the syncing of /etc/netns/global-vrf/resolv.conf to
> >          # /etc/resolv.conf requires 'ip netns exec global-vrf bash'.
> > -        self.send_xrnns('echo "# Cisco OpenDNS IPv4 nameservers" >> /etc/resolv.conf')
> > -        self.send_xrnns('echo "nameserver 208.67.222.222" >> /etc/resolv.conf')
> > -        self.send_xrnns('echo "nameserver 208.67.220.220" >> /etc/resolv.conf')
> > +        self.send_xrnns("echo '# Cisco OpenDNS IPv4 nameservers' >> /etc/resolv.conf")
>
> tee -a here too?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ios-xr/iosxrv-x64-vbox/pull/3/files/66cffbc15a8cb663494ffdaf3dd06ca6af25e05e..b0e8fdd00494b5d08432ec82b4a2d3f58700faa4#r69330827>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAYNnz1-l8dZxI7For8KuLfRmJyn7V0iks5qRUzagaJpZM4JCqR5>
> .
>